File: outergamesweb.client/src/app/interfaces/videojuego.ts
export interface Videojuego {
    idvideojuego: number;
    nombreVideojuego: string;
    descripcion: string;
    precio: number;
    cantidadStock: number;
    plataforma: string;
    genero1: string;
    genero2: string;
    genero3: string;
    imagen: string;
    cantidad?: number;
    precioTotal?: number;
}

File: OuterGamesWeb.Server/Models/Videojuego.cs
using System;
using System.Collections.Generic;

namespace OuterGamesWeb.Server.Models;

public partial class Videojuego
{
    public int Idvideojuego { get; set; }

    public string? NombreVideojuego { get; set; }

    public string? Descripcion { get; set; }

    public decimal? Precio { get; set; }

    public int? CantidadStock { get; set; }

    public string? Plataforma { get; set; }

    public string? Genero1 { get; set; }

    public string? Genero2 { get; set; }

    public string? Genero3 { get; set; }

    public string? Imagen { get; set; }

    public virtual ICollection<DetallesPedido> DetallesPedidos { get; set; } = new List<DetallesPedido>();
}
